src/ui/DynamicFacetManager/DynamicFacetManager.ts

```diff
@@ -159,7 +151,14 @@ export class DynamicFacetManager extends Component {
 
   private handleAfterComponentsInitialization() {
     this.childrenFacets = this.allDynamicFacets;
-    this.childrenFacets.forEach(dynamicFacet => (dynamicFacet.dynamicFacetManager = this));
+    this.childrenFacets.forEach(dynamicFacet => {
+      dynamicFacet.dynamicFacetManager = this;
+      this.containerElement.appendChild(dynamicFacet.element);
+    });
+
+    if (this.element.children.length > 1) {
+      this.logger.warn('DynamicFacetManager contains incompatible elements.');
```

Comment:
Is the warning necessary ? What happens if a end user leaves a "non compatible" element in there ?

Since we modified the code to not interact with non compatible element, is the warning still needed ?

edit: I see it moves them in the DOM. Might be good to add that information in the warning message (ie: it's important that warning and error message explains **why** something is an error/warning and what are the consequences).

Reply:
Suggestion:

```
DynamicFacetManager contains incompatible elements. Those elements may be moved in the DOM.
```

Reply:
Adding a solution:
`DynamicFacetManager contains incompatible elements. Those elements may be moved in the DOM.
        To prevent this warning, move those elements outside of the DynamicFacetManager.` 
@fbeaudoincoveo approved?

Reply:
Yep :)